src/main/java/frc/robot/commands/Climber2022DownCMD.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.commands;
+
+import frc.robot.logging.RobotLogManager;
+import frc.robot.subsystems.Climber2022;
+
+import java.util.function.Supplier;
+
+import org.apache.logging.log4j.Logger;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+
+public class Climber2022DownCMD extends CommandBase {
+    private final Climber2022 climber;
+    private final Supplier<Boolean> fullDown;
```

Comment:
What is `fullDown`? Is that asking the question, "Is the climber all the way down?" Usually I'd name that `isFullDown`, but a comment may be worthwhile here, too.